lang/cs/Org.Apache.REEF.Client/YARN/Parameters/SecurityTokenParameters.cs

```diff
@@ -19,13 +19,17 @@
 
 namespace Org.Apache.REEF.Client.YARN.Parameters
 {
-    [NamedParameter("Security token kind.", defaultValue: "NULL")]
+    //// TODO: [JIRA REEF-1887] Deprecated. Will remove.
+    [NamedParameter("Security token kind.", defaultValue: DefaultTokenKind)]
     public sealed class SecurityTokenKindParameter : Name<string>
     {
+        internal const string DefaultTokenKind = "NULL";
```

Comment:
This constant must have the same scope as the class. Please make it `public`

Reply:
Done